### ISSUE TITLE
Official Update - November 2020

### ISSUE THREAD

**Issue by jamesmontemagno** [jamesmontemagno/MediaPlugin#903]
[Xamarin.Essentials](https://github.com/xamarin/essentials?WT.mc_id=docs-github-jamont) 1.6 introduced official support for [picking/taking photos and videos](https://docs.microsoft.com/xamarin/essentials/media-picker?WT.mc_id=docs-github-jamont) with the new Media Picker API. 

This library has a lot of legacy code that is extremely hard to maintain and update to support the latest OSes without a major re-write. I will officially be archiving this library in December 2020 unless anyone from the community wants to adopt the project.

**Comment by MatthewKapteyn** [comment]
Dang - sad to hear the plans to archive this. While I'm certainly excited that Essentials is adding media picking support, they aren't nearly on the feature level of this plugin (for my purposes at least, the multi-picking feature and exif/metadata support).

I'd like to throw my hat in the ring for keeping the project going at least a little bit longer. 

That being said, I'd only be focusing on maintaining iOS and Android support as that's what I use in my own projects.  I have zero experience with Tizen and UWP. I just recently nixed them from my own fork.

Once Essentials reaches a feature level parity with this plugin, I don't see any reason to keep this plugin going either.

One way or the other, been very grateful for this plugin and want to say thanks for all the effort you've put into it over the years.

**Comment by lazmeister** [comment]
@MatthewKapteyn  I agree, Media Picker API is not even close to replacing this repo just yet and still buggy from what I can tell. I agree this plugin is getting old but I was hoping for the API to be more solid but its still in preview which is understable.

**Comment by Alextorres950325** [comment]
Essentials 1.6 has many bugs. better this plugin

**Comment by marcioqmaciel** [comment]
please, dont archive now

[Bug] MediaPicker CapturePhotoAsync - Orientation of photo rotated 90 degrees #1514
https://github.com/xamarin/Essentials/issues/1514

And with the coming of MAUI, no one knows if the Xamarin Forms team is working on the current project or if he died, at least no one from MS ever responded to the issues on the project's github.

**Comment by Kapusch** [comment]
Quick update: above mentioned issue may be fixed soon in MAUI with this incoming PR https://github.com/dotnet/maui/pull/14568
